data/companies.json

```diff
@@ -721,6 +721,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "Vertex Web Surf Pvt. Ltd.",
+      "establishedAt":1363198500,
```

Comment:
A little spacing between the key-value pair is needed to preserve the format. Also, I don't think the timestamp is correct.